### ISSUE TITLE
Fails to download monthly data

### ISSUE THREAD

**Issue by Hang-Duong** [ropensci/comtradr#89]
Hi! I am using "ct_get_bulk" to download monthly data for S3 and S4. However, I'm encountering an error for downloads prior to 2017. UNComtrade has confirmed that monthly data should be available from 2000. Could you advise me on how to check and resolve this error?

"Error in `ct_get_bulk()`:
! Probably no data for this combination of parameters
Run `rlang::last_trace()` to see where the error occurred.
Called from: signal_abort(cnd, .file)"

**Comment by datapumpernickel** [comment]
Hi Hang-Duong,

thanks for reporting this error. Interesting. 

On the [Comtrade Website](https://uncomtrade.org/docs/why-are-some-converted-datasets-not-accessible-in-the-ui/) at least for S4 it reports availability from 2007 onwards. Maybe they have a typo there and all are available from 2007 onwards? 

I would try sending them an email or try accessing the data directly through the Website with the "Try" function of the API to see if the error is with the package or with the API of UN Comtrade. I am currently travelling and do not have access to my login-credentials. 

Will also check this some time in the next two weeks. 

```
df_2007_s4 <- comtradr::ct_get_bulk(
  reporter = "DEU",
  commodity_classification = 'S4',
  start_date = 2008,
  end_date = 2008,
  frequency = 'M'
)
```

**Comment by Hang-Duong** [comment]
Thank you for your response. No further action is needed.

I attempted to obtain monthly data for Australia as the reporter, but Comtrade staff confirmed that such data is only available since 2017. This explains the error I encountered when using your packages. For other countries with available data, the codes work perfectly.

I also want to take this opportunity to thank your team for these useful packages. They are easy to apply and make accessing data from UN Comtrade very convenient.

Best regards,
Hang
